#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Fix byn_ER localedef breakage.
# DP: Related bugs: #246270, #257658.
# DP: Dpatch author: GOTO Masanori <gotom@debian.org>
# DP: Patch author: 
# DP: Upstream status: In CVS
# DP: Status Details: 
# DP: Date: 2004-07-14

PATCHLEVEL=0

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p$PATCHLEVEL < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p$PATCHLEVEL < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

# append the patch here and adjust the -p? flag in the patch calls.
--- locale/iso-639.def	2004-07-06 08:45:41.000000000 +0900
+++ locale/iso-639.def.new	2004-07-14 15:43:17.000000000 +0900
@@ -62,6 +62,7 @@
 DEFINE_LANGUAGE_CODE3 ("Bikol", bik, bik)
 DEFINE_LANGUAGE_CODE3 ("Bini", bin, bin)
 DEFINE_LANGUAGE_CODE ("Bislama", bi, bis, bis)
+DEFINE_LANGUAGE_CODE3 ("Blin", byn, byn)
 DEFINE_LANGUAGE_CODE ("Bosnian", bs, bos, bos)
 DEFINE_LANGUAGE_CODE3 ("Braj", bra, bra)
 DEFINE_LANGUAGE_CODE ("Breton", br, bre, bre)
